packages/rendermime/src/attachmentmodel.ts

```diff
@@ -0,0 +1,252 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+import {
```

Comment:
Please put the `@jupyterlab/...` imports before `@phosphor/...` (alphabetically)